packages/rum-core/src/domain/vital/vitalCollection.spec.ts

```diff
@@ -112,20 +148,35 @@ describe('vitalCollection', () => {
       })
     })
 
-    it('should create a vital from start API', () => {
-      const vital = vitalCollection.startDurationVital({
-        name: 'foo',
+    it('should create a vital from start API using name', () => {
+      vitalCollection.startDurationVital('foo', {
```

Comment:
**💬 suggestion:** ‏group vitalCollection tests together with a `describe('startVitalCollection'`

Reply:
Done

Reply:
Mmh what confused me is that we have tests for both `startDurationVital` and `vitalCollection.startDurationVital` at the same level. I would expect the module-level `startDurationVital` tests to be independent from tests needing `startVitalCollection`.